docs/docs/cmd/teams/channel/channel-set.md

```diff
@@ -10,11 +10,17 @@ m365 teams channel set [options]
 
 ## Options
 
-`-i, --teamId <teamId>`
-: The ID of the team where the channel to update is located
+`-i, --teamId [teamId]`
```

Comment:
In accordance with the Spec, could you remove the short `-i` here?

Could you also do this in the `channel get` command?